src/client/app/left/date-picker/date-picker.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, OnChanges, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  moduleId: module.id,
+  selector: 'sd-date-picker',
+  templateUrl: 'date-picker.component.html',
+  styleUrls: ['date-picker.component.css']
+})
+
+export class DatePickerComponent implements OnInit, OnChanges {
+  @Output() public datePicked = new EventEmitter();
+  @Input() maxInputDate: string;
+  public max: string = '2017-03-04';
```

Comment:
Okay, I understand that there are difference layers, however don't the variables always take on the same values in the class?  I've tested removing it on my branch and it seems to work fine?